File: keda-azure-blob-storage/controllers/homeController.js
import express from "express"

const router = express.Router()

export const homeController = router.get("/", (_, res) => {
    try {
        res.json({ msg: "container-apps-scaling-examples-keda-blog-storage"})
    } catch (error) {
        console.error(error)
    }
})

File: keda-azure-service-bus/src/main/java/com/servicebus/keda/Controllers/HomeController.java
package com.servicebus.keda.Controllers;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HomeController {
    String message = "container-apps-scaling-examples-keda-service-bus";

    @GetMapping("/")
    public String index() {
        return message;
    }
}
